src/models/OldMainMethod.py

```diff
@@ -0,0 +1,73 @@
+        # All lists containt list of lists of features in one hot encoded format ready to append to
+        # cleaned set
+
+        # Description: consisting of 5 features existence of words below in the description:
+        # 'excellent', 'graduate', 'immediate', 'junior', 'urgent'
+        onehot_encoded_desc_words = get_one_hot_encoded_words(self.train_description_feature)
+        # Contract type: One hot encoded, 3 features: part time, full time, *is empty*
+        onehot_encoded_contract_type = get_one_hot_encoded_feature(self.train_contract_type_feature)
+        # Contract time: One hot encoded, 3 features: permanent, contract, *is empty*
+        onehot_encoded_contract_time = get_one_hot_encoded_feature(self.train_contract_time_feature)
+
+        # Train binary encoded features - company, source, town, region
+        # Company name: Binary encoded
+        train_binary_encoded_company = get_binary_encoded_feature(self.train_company_feature)
+        # Source name: Binary encoded
+        train_binary_encoded_source = get_binary_encoded_feature(self.train_source_name_feature)
+        # Town: Binary encoded
+        cleaned_binary_encoded_town = get_binary_encoded_feature(updated_town_feature)
+        # Region: Binary encoded
+        cleaned_binary_encoded_region = get_binary_encoded_feature(updated_region_feature)
+
+        # get stemmed/sorted job title and job title modifiers
+
+        processed_job_titles, processed_job_modifiers = get_stemmed_sentences(
+            self.train_raw_data[['Title']]
+        )
+
+        train_binary_encoded_job_titles = get_binary_encoded_feature(processed_job_titles)
+        train_binary_encoded_job_modifiers = get_binary_encoded_feature(processed_job_modifiers)
+
+        keywords = get_rake_keywords(self.train_description_feature)
+        # one_hot_encoded_features are a list of 5 lists consisting of 1s and 0s
+        # If a word appears in the description, this kind of representation will
+        # make it feasible for machine learning
+        # We will need to append those lists on final data frame when everything is cleaned
+
+        # get top k terms with the highest idf score (this is atm not very useful
+        # but it will be used for the tf.idf similarity stuff
+        top_keywords = get_top_idf_features(self.train_description_feature, 5)
+
+        # no-op functions to avoid pep8 "unused variable" error
+        isinstance(onehot_encoded_desc_words, int)
+        isinstance(onehot_encoded_contract_type, int)
+        isinstance(onehot_encoded_contract_time, int)
+        isinstance(train_binary_encoded_company, int)
+        isinstance(train_binary_encoded_source, int)
+        isinstance(cleaned_binary_encoded_town, int)
+        isinstance(cleaned_binary_encoded_region, int)
+        isinstance(train_binary_encoded_job_titles, int)
+        isinstance(train_binary_encoded_job_modifiers, int)
+        isinstance(keywords, int)
+        isinstance(top_keywords, int)
```

Comment:
This isn't great - we can just mute the error on Travis (I'll go ahead and do that if you remove these lines)